monasca/templates/agent-daemonset.yaml

```diff
@@ -121,4 +121,12 @@ spec:
               value: {{ .Values.agent.insecure | quote }}
             - name: HOSTNAME_FROM_KUBERNETES
               value: "true"
+            {{- if .Values.agent.cadvisor.enable_minimum_whitelist }}
```

Comment:
Do we need these changes in agent-deployment too?

Reply:
nope they dont run these plugins